src/components/box/Box.tsx

```diff
@@ -4,7 +4,9 @@ import { COLOR, COLORS, TINTS } from '../../constants';
 import theme from './theme.css';
 
 const margins = [-8, -7, -6, -5, -4, -3, -2, -1, 0, 1, 2, 3, 4, 5, 6, 7, 8] as const;
+export type Margin = typeof margins[number];
```

Comment:
Q: Won't it be more convenient to have `Margin` and `Padding` in separate file? just from hierarchy POV

Reply:
I inlined them because they were defined inline before will need to keep an eye on components that do not go through the Box component to set margins or paddings, but I think most components use this base component at the moment